### DISCUSSION TITLE
HTTP Request Node Type

### DISCUSSION THREAD

**Discussion by biapar** [chrisjpatty/flume#245] [Q&A]
Are there HTTP request node type?

**Comment by idling-mind** [comment]
As far as i know, flume doesn't come with any pre-populated node/port types. You have to create nodes and ports on your own.

**Comment by biapar** [comment]
Is there an example?

**Reply by idling-mind** [reply]
https://flume.dev/docs/basic-config#create-your-first-node

**Comment by biapar** [comment]
I already read it in but I don’t see how to Made complex operation on input o get external data to elaborate and push the data to the output.

**Reply by idling-mind** [reply]
I think, that's more of a javascript/react question than a question for this library. You'll have to add your logic on how you resolve each node in your root engine. https://flume.dev/docs/root-engine#resolving-nodes
